public/javascripts/SVLabel/src/SVLabel/data/Form.js

```diff
@@ -255,13 +258,20 @@ function Form (labelContainer, missionModel, missionContainer, navigationModel,
                     var taskId = result.audit_task_id;
                     task.setProperty("auditTaskId", taskId);
                     svl.tracker.setAuditTaskID(taskId);
-
+                    
                     // If the back-end says it is time to switch to validations, then do it immediately (mostly to
                     // prevent turkers from modifying JS variables to prevent switching to validation).
                     if (result.switch_to_validation) window.location.replace('/validate');
 
                     // If a new mission was sent and we aren't in onboarding, create an object for it on the front-end.
                     if (result.mission && !svl.isOnboarding()) missionModel.createAMission(result.mission);
+
+                    // Update the time the database performed the query to update our local task priorities with
```

Comment:
This comment is just overly wordy and kinda confusing. Like I know what's going on, and I still had to read it 3+ times to get what you were going for here lol